lib/traject/config/folio_config.rb

```diff
@@ -165,7 +165,11 @@ def holdings(record, context)
 # guard against dates with 'u' coming out of MARC 008 fields
 to_field 'date_cataloged' do |record, accumulator|
   timestamp = record.instance['catalogedDate']
-  accumulator << Time.parse(timestamp).utc.at_beginning_of_day.iso8601 if timestamp =~ /\d{4}-\d{2}-\d{2}/
+  begin
+    accumulator << Time.parse(timestamp).utc.at_beginning_of_day.iso8601 if timestamp =~ /^\d{4}-\d{2}-\d{2}/
+  rescue StandardError
```

Comment:
Can this be `ArgumentError` instead?

Reply:
🤷‍♂️ i don't feel strongly either way

Reply:
As a reader, ArgumentError helps me understand the intention of the code without needing to refer to the commit message.